core/app/c/[communitySlug]/fields/FieldForm.tsx

```diff
@@ -112,14 +137,24 @@ const SchemaSelectField = ({ form, isDisabled }: { form: FormType; isDisabled?:
  * The user can overwrite this value via the text input. The value here does not have
  * the community prepended—the community is automatically prepended in handleSubmit
  */
-const SlugField = ({ form, communitySlug }: { form: FormType; communitySlug: string }) => {
+const SlugField = ({
+	form,
+	communitySlug,
+	readOnly,
+}: {
+	form: FormType;
+	communitySlug: string;
+	readOnly?: boolean;
+}) => {
 	const { watch, setValue } = form;
 
 	const watchName = watch("name");
 
 	useEffect(() => {
-		const autoSlug = slugifyString(watchName);
-		setValue("slug", autoSlug);
+		if (!readOnly) {
```

Comment:
we got away with this before because I had just hidden this field in the Edit modal, but Katy and I agreed it makes more sense to still show these fields, but have them "disabled" instead